src/v/config/configuration.cc

```diff
@@ -436,6 +436,26 @@ configuration::configuration()
       "truncation if any of the replicas has more data than the majority.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , raft_max_inflight_follower_append_entries_requests(
+      *this,
+      "raft_max_inflight_follower_append_entries_requests",
+      "Maximum number of append entries requests, that were sent to a node "
```

Comment:
Suggestion for changes to description. @mmaslankaprv, please check for accuracy:

The maximum number of append entry requests that may be sent from raft groups on a Seastar shard to the current node. Previously, the maximum number of requests was set per raft group."

Reply:
Here it is crutial to say that the limit is for the requests that are pending the suggested description doesn't state that

Reply:
The maximum number of append entry requests that may be sent from raft groups on a Seastar shard to the current node and are awaiting a reply. This property replaces the [`raft_max_concurrent_append_requests_per_follower`](https://docs.redpanda.com/current/reference/properties/cluster-properties/#raft_max_concurrent_append_requests_per_follower), which was deprecated in 24.2.

@mmaslankaprv. Is this accurate now?

@Deflaimun - Not sure how we handle deprecations of properties? It looks like we add them to this table (https://docs.redpanda.com/current/upgrade/deprecated/) but do they get marked as deprecated for a period of time before they are removed from the list of properties.